pkg/testcase/certrotate.go

```diff
@@ -0,0 +1,109 @@
+package testcase
+
+import (
+	"fmt"
+
+	"github.com/rancher/distros-test-framework/factory"
+	"github.com/rancher/distros-test-framework/shared"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+)
+
+func TestCertRotate() {
+	// Please refer to docs/testing.md for instructions on setup for this test
+
```

Comment:
no need for this first empty line + no need to ref to docs
this ref should be documented in some doc, readme , doesn matter, but not in code .

Reply:
fixed